components/SearchResults/SearchResults.js

```diff
@@ -32,7 +32,10 @@ const SearchResults = ({ searchResults }) => {
   return (
     <SearchResultsContainer>
       {filteredResults.map((result) => (
-        <Link href={`/album/${result.id}`} key={result.id}>
+        <Link
+          href={`/album/${result.id}?coverImage=${result.cover_image}`}
+          key={result.id}
+        >
```

Comment:
It's a hack to pass down the cover image separately. Please avoid doing so.

 ```suggestion
        <Link
          href={`/album/${result.id}`}
          key={result.id}
        >
```